server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJmapTestRule.java

```diff
@@ -42,17 +47,26 @@ public static CassandraLdapJmapTestRule defaultTestRule() {
             AggregateGuiceModuleTestRule.of(new EmbeddedElasticSearchRule(), new DockerCassandraRule()));
     }
 
-    private GuiceModuleTestRule guiceModuleTestRule;
+    private final TemporaryFolder temporaryFolder;
+    private final GuiceModuleTestRule guiceModuleTestRule;
 
     public CassandraLdapJmapTestRule(GuiceModuleTestRule... guiceModuleTestRule) {
+        TempFilesystemTestRule tempFilesystemTestRule = new TempFilesystemTestRule();
+        temporaryFolder = tempFilesystemTestRule.getTemporaryFolder();
         this.guiceModuleTestRule =
             AggregateGuiceModuleTestRule
                 .of(guiceModuleTestRule)
-                .aggregate(new TempFilesystemTestRule());
+                .aggregate(tempFilesystemTestRule);
     }
 
-    public GuiceJamesServer jmapServer(String ldapIp, Module... additionals) {
-        return new GuiceJamesServer()
+    public GuiceJamesServer jmapServer(String ldapIp, Module... additionals) throws IOException {
+        String workingDir = temporaryFolder.newFolder().getAbsolutePath();
+        Configuration configuration = Configuration.builder()
+            .workingDirectory(workingDir)
+            .configurationPath(FileSystem.CLASSPATH_PROTOCOL)
```

Comment:
`configurationPath(FileSystem.CLASSPATH_PROTOCOL)` should have a helper method like `configurationFromClassPath()` to make starting a James server easier in tests?

Reply:
CLASSPATH is not available in core.

Maybe this is a strong indicator that `Configuration` should be part of `filesystem-api`. WDYT?